uswid/patch.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+# (c) Copyright 2025 HP Development Company, L.P.
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+from typing import List, Any, Optional
+from .purl import uSwidPurl
+
+class uSwidPatch:
+    """Represents a patch"""
+
+    def __init__(
+        self,
+        patch_type: str = None,  # e.g. 'backport', 'cherry-pick', 'security'
+        patch_url: Optional[uSwidPurl] = None,    # URL to the patch/diff
+        description: Optional[List[dict]] = None,  # List the details of the patch
```

Comment:
what is this supposed to be a dict of? Ideologically, isn't the description just a string? We can parse and build any format we want when going to/from CycloneDX, but I think dict of any:any is painting ourself into a corner for SPDX support.

Reply:
i.e. you could add a .references str here -- or anything else needed for CycloneDX -- of we could even have a new uSwidPatchDescription object and have a list of those -- but maybe the latter is overengineering.